stepconf/stepconf.go

```diff
@@ -195,15 +195,18 @@ func ValidateRangeFields(valueStr, constraint string) error {
 	}
 	min, err := parseValueStr(constraintMin)
 	if err != nil {
-		return fmt.Errorf("failed to parse min value: %s", err)
+		return fmt.Errorf("failed to parse min value %s: %s", constraintMin, err)
 	}
 	max, err := parseValueStr(constraintMax)
 	if err != nil {
-		return fmt.Errorf("failed to parse max value: %s", err)
+		return fmt.Errorf("failed to parse max value %s: %s", constraintMax, err)
+	}
+	if valueStr == "" {
```

Comment:
Please see my comment.

Reply:
Updated code.

Reply:
please move this empty check to the first line of the block to exit-fast

Reply:
Moved to first line.